workspaces/ui/src/routes.js

```diff
@@ -1,65 +1,44 @@
 import React from 'react';
-import { Redirect, Switch, Route } from 'react-router-dom';
+import {Redirect, Switch, Route} from 'react-router-dom';
 import Welcome from './components/onboarding/Welcome';
 import ExampleSessionsLoader from './components/loaders/ExampleSessionsLoader.js';
 import LocalLoader from './components/routes/local';
-import { routerPaths } from './RouterPaths'
+import {useRouterPaths} from './RouterPaths';
 import SharedLoader from './components/loaders/SharedLoader';
-import { ExampleTestingServiceLoaderComponent } from './components/loaders/TestingDashboardLoader';
+import TestingDashboardLoader from './components/loaders/TestingDashboardLoader';
+import ApiSpecServiceLoader from './components/loaders/ApiLoader';
+import {CaptureManagerPage} from './components/diff/v2/CaptureManagerPage';
+import {DocsPage} from './components/docs/DocsPage';
 
+export function ApiRoutes(props) {
+  const routerPaths = useRouterPaths();
```

Comment:
I keep getting confused with stuff being named "Api*", as I instantly go "wait, are we serving an API with this?". Only to remember that we're making an app about documenting APIs. I think we could do with better names for these different parts of the entire system / targets 😅.

Reply:
Agreed. Let's iterate on a glossary in Notion and then get it into the codebase / public docs